pkg/process/info.go

```diff
@@ -162,7 +165,15 @@ func NewInfoManager(
 		mapManager:       mm,
 		debuginfoManager: dim,
 		labelManager:     lm,
+
+		uploadJobQueue: make(chan *uploadJob, 128),
+		uploadJobPool: &sync.Pool{
```

Comment:
Was this pool really necessary? Couldn't we have just used a channel that passes value structs?

Reply:
happy to do that after we release, it's minor at this point